### DISCUSSION TITLE
Unity Common Package missing tag

### DISCUSSION THREAD

**Discussion by Simse92** [reown-com/appkit#2931] [General]
Hey guys. 

I tried to install the unity package via openupm and it failed cause your common package has a missing tag.
https://openupm.com/packages/com.reown.core.common/?subPage=readme

Simon

**Comment by rtomas** [comment]
cc @skibitsky

**Comment by skibitsky** [comment]
Thanks for flagging this, @Simse92. The common package is now available on OpenUPM.

**Reply by Simse92** [reply]
Awesome. Thx for the fast response.
Closed
